src/main/java/org/folio/circulation/resources/ChangeDueDateResource.java

```diff
@@ -96,12 +102,13 @@ private LoanAndRelatedRecords unsetDueDateChangedByRecallIfNoOpenRecallsInQueue(
     
     RequestQueue queue = loanAndRelatedRecords.getRequestQueue();
     Loan loan = loanAndRelatedRecords.getLoan();
-    log.info("Loan " + loan.getId() + " prior to flag check: " + loan.asJson().toString());
-    if (loan.wasDueDateChangedByRecall() && !queue.hasOpenRecalls()) {
-      log.info("Loan " + loan.getId() + " registers as having due date change flag set to true and no open recalls in queue.");
+    log.info("Loan {} prior to flag check: {}", loan.getId(), loan.asJson().toString());
+    if (loan.wasDueDateChangedByRecall() && !queue.hasOpenRecalls() && (Objects.isNull(loanAndRelatedRecords.getTlrSettings())
```

Comment:
No need to add additional conditions here. Everything should work correctly as is.

Reply:
The new conditions are removed, thank you!